packages/web3-constants/evm/token-list.json

```diff
@@ -17,6 +17,8 @@
         "Celo": ["https://tokens.r2d2.to/latest/42220/tokens.json"],
         "Fantom": ["https://tokens.r2d2.to/latest/250/tokens.json"],
         "Aurora": ["https://tokens.r2d2.to/latest/1313161554/tokens.json"],
-        "Aurora_Testnet": []
+        "Aurora_Testnet": [],
+        "Conflux": [],
```

Comment:
Strange! It has value on the `develop` branch. Do you mind merging the `develop` branch?